packages/health-check/src/checks/critical.js

```diff
@@ -25,24 +28,45 @@ async function uploadCheck(done) {
     data.ip = error?.response?.ip ?? null;
   }
 
-  done({
-    name: "upload_file",
-    time: calculateElapsedTime(time),
-    ...data,
-  });
+  done({ name: "upload_file", time: calculateElapsedTime(time), ...data });
+}
+
+// websiteCheck checks whether the main website is working
+async function websiteCheck(done) {
+  return genericAccessCheck("website", process.env.SKYNET_PORTAL_API, done);
 }
 
 // downloadCheck returns the result of downloading the hard coded link
 async function downloadCheck(done) {
+  const url = await skynetClient.getSkylinkUrl(exampleSkylink);
+
+  return genericAccessCheck("skylink", url, done);
+}
+
+// skylinkSubdomainCheck returns the result of downloading the hard coded link via subdomain
+async function skylinkSubdomainCheck(done) {
+  const url = await skynetClient.getSkylinkUrl(exampleSkylink, { subdomain: true });
+
+  return genericAccessCheck("skylink_via_subdomain", url, done);
+}
+
+// skylinkSubdomainCheck returns the result of downloading the hard coded link via subdomain
+async function handshakeSubdomainCheck(done) {
+  const url = await skynetClient.getHnsUrl("note-to-self", { subdomain: true });
+
+  return genericAccessCheck("hns_via_subdomain", url, done);
+}
+
+async function accountHealthCheck(done) {
```

Comment:
Follow Up: docstrings for other functions.

Reply:
added